README.md

```diff
@@ -36,71 +36,59 @@ It supports several backends ([Docker](https://www.docker.com/), [Swarm mode](ht
 
 ## Overview
 
-Imagine that you have deployed a bunch of microservices on your infrastructure. You probably used a service registry (like etcd or consul) and/or an orchestrator (swarm, Mesos/Marathon) to manage all these services.
-If you want your users to access some of your microservices from the Internet, you will have to use a reverse proxy and configure it using virtual hosts or prefix paths:
+Imagine that you have deployed a bunch of microservices with the help of an orchestrator (like Swarm or Mesos/Marathon) or a service registry (like etcd or consul).
```

Comment:
should mention k8s. It is ballooning in usage ;)

Reply:
you're right.